components/modules/node.c

```diff
@@ -149,6 +150,36 @@ static int node_dsleep (lua_State *L)
   return 0;
 }
 
+static void add_int_field( lua_State* L, lua_Integer i, const char *name){
+  lua_pushinteger(L, i);
+  lua_setfield(L, -2, name);
+}
+
+static void add_string_field( lua_State* L, const char *s, const char *name) {
+  lua_pushstring(L, s);
+  lua_setfield(L, -2, name);
+}
+
+static int node_info( lua_State* L ){
+  const char* options[] = {"sw_version", "default", NULL};
```

Comment:
Do you think it would make sense to support the same options as with the ESP8266 but return an empty result for every option not yet implemented/applicable? Hhmm, maybe just a brain fart, don't know.

`const char* options[] = {"lfs", "hw", "sw_version", "build_config", "legacy", NULL};`